docs/time/performance.inc.rst

```diff
@@ -10,3 +10,52 @@ Performance Tips
 
 Here we provide some tips and tricks for how to optimize performance of code
 using `astropy.time`.
+
+Light Travel Times
+------------------
+
+Listed below are two approaches to calculating light travel times for tens of
+thousands of sources in a degree patch of the sky. The first approach calculates
+the travel times without iteration while the second calculates using iteration.
+
+
+>>> import numpy as np
+>>> import astropy.coordinates as coord
+>>> import astropy.units as u
+>>> from astropy.time import Time
+
+>>> ra = np.random.normal(0.0, 1.0, 50000)
+>>> dec = np.random.normal(0.0, 1.0, 50000)
+>>> coos = coord.SkyCoord(ra, dec, unit=u.deg)
+>>> observatory = coord.EarthLocation.from_geocentric(5327448.9957829, -1718665.73869569, 3051566.90295403, unit='m') #Lapalma Observatory
```

Comment:
Perhaps you can put the end-of-line comment above in it's own line, and perhaps note that 
```coord.EarthLocation.of_site('lapalma')``` would also work if you have internet connection

Reply:
I will do that!